### DISCUSSION TITLE
ChampR的功能建议

### DISCUSSION THREAD

**Discussion by sugarnine9** [cangzhang/ChampR#163] [General]
距离上一次提建议也过了很长时间了，但是这款软件好像最近被配置符文的失效问题所困扰着，功能上没有太大的更新。 今天看到一个大佬自制的软件（  http://lol078.com/  ），有些功能很有趣：
1、自动接受对局。这个功能应该很常见，也很好做。 有时候在玩手机，或者去干其他事情的时候，就会忘记点确认。或者说，可以设置一个时间限制，倒数5秒的时候自动确认。 
2、自动查询队友或对手的战绩、 KD和 胜率等。方便队友知道敌我之间的差距。 这一功能对打野比较有用，对个人对线也有一些作用。
3、自动根据位置自动选用/禁用英雄。
下面是我自己想的一些功能：
1、OPGG英雄等级排行。这个在匹配或者排位的时候比较有用，就是说在没有选择英雄之前，软件有一个弹窗，显示上单、打野、中单、下路、辅助这5个位置的opgg英雄等级排行，方便玩家看当前版本热门英雄。 

可能这些功能偏离了这款软件的初心——英雄符文一键配置，但我觉的是一个不错的发展方向。如果大佬觉得有意向的话，可以看看。

最后，再次感谢大佬的辛勤付出。祝你生活愉快。

**Comment by MTKmin** [comment]
我只是单纯的想要一个自动接受就可以了 然后加上天赋 TGP里的天赋简直不能看  太辣眼睛 而且我只玩大乱斗  很多次都有人秒  有的时候倒水去 发现秒了=。=  下一次对局没有点到接受=。=
